gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyDedupReducer.java

```diff
@@ -61,6 +62,11 @@ protected void reduce(OrcKey key, Iterable<OrcValue> values, Context context)
     int valueHash = 0;
 
     for (OrcValue value : values) {
+      recordCounter++;
+      if (recordCounter % 1000 == 0) {
+        log.info("Reduced %s values for key %s", recordCounter, key);
```

Comment:
Seems like the log does not looks correct after you move it to class level. 
You also want to print one log when we start to reduce the first record?